pyaedt/hfss.py

```diff
@@ -4285,9 +4476,11 @@ def create_scattering(
         sweep_name : str, optional
              Name of the sweep. The default is ``None``.
         port_names : list, optional
-             List of port names. The default is ``None``.
+             List of port names. The first index, i, in S[i,j].
+             The default is ``None``. (include only self-terms)
```

Comment:
```suggestion
             List of port names. The first index, i, in S[i,j].
             The default is ``None``. (include only self-terms)
```
This description and the next should be written out clearly in complete sentences.